src/components/WebexMeetingControl/WebexMeetingControl.stories.js

```diff
@@ -49,3 +49,13 @@ stories.add('disabled icon', () => (
     </WebexMeetingControls>
   </WebexDataProvider>
 ));
+
+stories.add('multiple icons', () => (
+  <WebexDataProvider adapter={webexAdapter}>
+    <WebexMeetingControls meetingID="localMediaWithSharing">
```

Comment:
This meeting ID no longer exists, so the story is not displaying properly

Reply:
use localMedia instead